specifications/api/swagger.yaml

```diff
@@ -1926,6 +1937,17 @@ components:
         payment_script_template: *scriptTemplate
         delegation_script_template: *scriptTemplate
 
+    ApiSharedWalletPatchData: &ApiSharedWalletPatchData
+      type: object
+      required:
+        - cosigner
+        - account_public_key
+        - script_template_update
```

Comment:
Do we want to have ability to update wallet's name? (Perhaps that would be job for different ep, like `update metadata`?)

Reply:
I will do this in either this or next PR (probably in the next as this PR got really big)